tools/pbxproj_generators/project_prefix/src/Generator/Arguments.swift

```diff
@@ -0,0 +1,39 @@
+import ArgumentParser
+import Foundation
+import PBXProj
+
+extension Generator {
+    struct Arguments: ParsableArguments {
+        @Argument(
+            help: """
+Path to where the 'project_prefix' '.pbxproj' partial should be written.
```

Comment:
Everywhere I mention `.pbxproj` I can probably say `PBXProj` instead. A downside to that is besides for the filename it's never called that (unlike the other nested types like `PBXProject`). If people search though, they will find it called that in `XcodeProj` and the like.

```suggestion
Path to where the 'project_prefix' '.pbxproj' partial should be written.
```